README.md

```diff
@@ -26,4 +24,14 @@ The plan for Hazel is two-fold: to create a powerful 3D engine, but also to serv
 - Integrated 3rd party 2D and 3D physics engine
 - Procedural terrain and world generation
 - Artificial Intelligence
-- Audio system
\ No newline at end of file
+- Audio system
+## Short term goals :
+By the end 2020, we want to make a game, using the Hazel game engine.
+
+*note this is subject to change at any time.*
```

Comment:
First of all, add an empty line before the title.

Secondly, the note should be more or less explicit to the short term goal, hence why I suggest putting it in front of the actual text of the short term goal.

At last, as discussed on discord, you should write it in more than a single line.. maybe also reference to one of the videos where he explains this goal.

```suggestion

## Short term goals :
*note this is subject to change at any time.*
By the end 2020, we want to make a game, using the Hazel game engine.
```